dozer/cogs/firstqa.py

```diff
@@ -100,6 +101,49 @@ async def frcqa(self, ctx: DozerContext, question: int):
     """
 
 
+    @commands.hybrid_command(name = "frcrule", pass_context = True)
+    @bot_has_permissions(embed_links = True)
+    @app_commands.describe(rule = "The rule number")
+    async def frcrule(self, ctx: DozerContext, rule: str):
+        letter_part = ''.join([char for char in rule if char.isalpha()])
+        number_part = ''.join([char for char in rule if char.isdigit()])
+        embed = discord.Embed(
+            title=f"Rule {letter_part.upper()}{number_part}",
+            url=f"https://frc-qa.firstinspires.org/manual/rule/{letter_part.upper()}/{number_part}",
```

Comment:
We shouldn't display any data before doing the validation of it. This URL could be broken if it doesn't exist or adhere to the regex we have. The error state and happy path embeds should be entirely different (and ideally the unhappy path embed is ephemeral like you had earlier).